modules/luci-mod-status/luasrc/controller/admin/status.lua

```diff
@@ -5,6 +5,9 @@
 module("luci.controller.admin.status", package.seeall)
 
 function index()
+	
+	local hasAccess = nixio.fs.access
```

Comment:
Is this necessary?

Reply:
I would like prefere useing the following checks:

local has_wireless = fs.access("/usr/bin/wireless")
local has_fstab = fs.access("/etc/config/fstab")
local has_block = fs.access("/sbin/block")

This will be used in LuCI